src/test/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensionSpec.scala

```diff
@@ -528,11 +528,62 @@ class GoogleExtensionSpec(_system: ActorSystem) extends TestKit(_system) with Fl
     val firstKey = runAndWait(googleExtensions.getPetServiceAccountKey(defaultUser, googleProject))
 
     //remove the key we just created
-    runAndWait(googleExtensions.removePetServiceAccountKey(defaultUserId, googleProject, ServiceAccountKeyId(firstKey)))
```

Comment:
`firstKey` is not a key id so this was incorrect. It only worked because the mock iam dao ignored the key id.